RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Live.swift

```diff
@@ -219,6 +251,9 @@ extension CloudBackupClient {
 				try await fetchAllProfileRecords(headerOnly: true)
 					.map(getProfileHeader)
 					.filter(\.isNonEmpty)
+			},
+			reclaimProfile: {
+				try await profileStore.claimOwnershipOfProfile()
```

Comment:
Isn't this basically a noop? I mean with new backup system, there is no way that current profile can have a different `lastUsedOnDeviceId`. Am I missing something?

Reply:
No you are right. I incorrectly thought that the local profile wasn't claimed automatically when importing from a file, so this is not needed.